tutorials/create-events-dataset.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: Creating the Events Dataset
+---
+## Purpose
+
+This lesson shows you how to create a dataset suitable for storing standard event records, as defined in [The Unified Logging Infrastructure for Data Analytics at Twitter][paper]. You define a dataset schema, a partition strategy, and a URI that specifies the storage scheme.
```

Comment:
I think this should differentiate between the skills that are demonstrated and the purpose in the context of the tutorials. The skill is to use a given Avro schema and partition strategy to create a dataset. The context for the tutorials is that this module has the reader create a dataset for the standard event records that are used by subsequent modules. The next modules should also be linked somewhere in the context section.